src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -261,6 +265,18 @@ void FullyConnected::getSupportedDescriptors() {
 
     inDims = isDynamicNode() ? makeDummyInputDims() : getInputShapeAtPort(DATA_ID).getStaticDims();
     outDims = isDynamicNode() ? makeDummyOutputDims(inDims) : getOutputShapeAtPort(0).getStaticDims();
+#ifdef OV_CPU_WITH_MLAS
```

Comment:
Could you please isolate all the new ifdef entries with the separate functions.
Otherwise the code becomes hard to read.

Reply:
Great suggestion, will add 2 methods to compute GEMM.
1. WeightPrepack
2. GemmCompute